app.js

```diff
@@ -2,108 +2,99 @@
 if (process.env.NODE_ENV !=='production'){ // if the environment is development include env file
     require('dotenv').config(); // this will load all environment processes
 }
-
-// Dependencies
+/**
+ * -----------------DEPENDENCIES-------------------
+ */
 const express = require('express');
 const path = require('path');
-const passport = require('passport');
-const bodyParser = require('body-parser');
-const LocalStrategy = require('passport-local');
+const mysql = require('mysql');
 const bcrypt = require('bcrypt');
-const session = require('express-session');
+const bodyParser = require('body-parser');
 
+/**
+ * --------------GENERAL SETUP---------------------
+ */
 const app = express();
 
-// // File imports
-const initializePassport = require('./configure/passport-configure');
-initializePassport(
-    passport, // configures passport  
-    email => users.find(user => user.email === email), // set user to user.email and make it set to email
-    id => user.find(user => user.id === id)
-);
-
-// Test User Authentication
-let users = [];
-
-// This allows input to be accessed through requests and POSTS
-app.use(express.urlencoded({ extended: false }))
-app.use(session({
-    secret: process.env.SESSION_SECRET,
-    resave: false, // this does not change session variable if nothing is changed
-    saveUninitalized: false // does not save empty value into the session
-}))
-app.use(passport.initialize()) // sets up initial basic configuration of passport
-app.use(passport.session()) // itneracts with the app.use session above
-app.use(express.json()) // TESTING PURPOSES
-
-
-// Routes
-app.get('/', (req, res) => {
-    res.sendFile(path.join(__dirname + '/views/home.html'));
-}); 
+const users = [];
+
+app.use(express.json()) // Allows file to read JSON 
 
-// Register routes
-app.get('/test', (req, res) => {
-    res.json(users); // return users array
+
+/**
+ * --------------DATABASE CONFIG-----------------
+ */
+let connection = mysql.createConnection({
+    host: "localhost",
+    user: "rs-217",
+    password: "$udoPowers37",
+    database:'node_app'
```

Comment:
We will eventually use .env variables after you're done testing and reference it using process.env.ENV_VAR

Reply:
Roger that!